yb-voyager/src/tgtdb/yugabytedb_test.go

```diff
@@ -193,54 +193,99 @@ func TestYugabyteGetNonEmptyTables(t *testing.T) {
 	testutils.AssertEqualNameTuplesSlice(t, expectedTables, actualTables)
 }
 
-func TestGetPrimaryKeyConstraintName(t *testing.T) {
+func TestGetPrimaryKeyConstraintNames(t *testing.T) {
 	testYugabyteDBTarget.ExecuteSqls(
 		`CREATE SCHEMA test_schema;`,
 		`CREATE TABLE test_schema.foo (
-			id INT PRIMARY KEY,
-			name TEXT
-		);`,
+            id INT PRIMARY KEY,
+            name TEXT
+        );`,
 		`CREATE TABLE test_schema.bar (
-			id INT,
-			name TEXT,
-			CONSTRAINT bar_primary_key PRIMARY KEY (id)
-		);`,
+            id INT,
+            name TEXT,
+            CONSTRAINT bar_primary_key PRIMARY KEY (id)
+        );`,
 		`CREATE TABLE test_schema.baz (
-			id INT,
-			name TEXT
-		);`,
+            id INT,
+            name TEXT
+        );`,
 		`CREATE TABLE test_schema."CASE_sensitive" (
-			id INT,
-			name TEXT,
-			CONSTRAINT "CASE_sensitive_pkey" PRIMARY KEY (id)
-		);`,
+            id INT,
+            name TEXT,
+            CONSTRAINT "CASE_sensitive_pkey" PRIMARY KEY (id)
+        );`,
+		// Partitioned table examples
+		`CREATE TABLE test_schema.emp (
+            emp_id   INT,
+            emp_name TEXT,
+            dep_code INT,
+            PRIMARY KEY (emp_id)
+        ) PARTITION BY HASH (emp_id);`,
+		`CREATE TABLE test_schema.emp_0 PARTITION OF test_schema.emp FOR VALUES WITH (MODULUS 3, REMAINDER 0);`,
+		`CREATE TABLE test_schema.emp_1 PARTITION OF test_schema.emp FOR VALUES WITH (MODULUS 3, REMAINDER 1);`,
+		`CREATE TABLE test_schema.emp_2 PARTITION OF test_schema.emp FOR VALUES WITH (MODULUS 3, REMAINDER 2);`,
+
+		`CREATE TABLE sales_region (id int, amount int, branch text, region text, PRIMARY KEY(id, region)) PARTITION BY LIST (region);`,
+		`CREATE TABLE London PARTITION OF sales_region FOR VALUES IN ('London');`,
+		`CREATE TABLE Sydney PARTITION OF sales_region FOR VALUES IN ('Sydney');`,
+		`CREATE TABLE Boston PARTITION OF sales_region FOR VALUES IN ('Boston');`,
+
+		// Multi level partitioning
+		`CREATE TABLE customers (id INTEGER, statuses TEXT, arr NUMERIC, PRIMARY KEY(id, statuses, arr)) PARTITION BY LIST(statuses);`,
+
+		`CREATE TABLE cust_active PARTITION OF customers FOR VALUES IN ('ACTIVE', 'RECURRING','REACTIVATED') PARTITION BY RANGE(arr);`,
+		`CREATE TABLE cust_other  PARTITION OF customers DEFAULT;`,
+
+		`CREATE TABLE cust_arr_small PARTITION OF cust_active FOR VALUES FROM (MINVALUE) TO (101) PARTITION BY HASH(id);`,
+		`CREATE TABLE cust_part11 PARTITION OF cust_arr_small FOR VALUES WITH (modulus 2, remainder 0);`,
+		`CREATE TABLE cust_part12 PARTITION OF cust_arr_small FOR VALUES WITH (modulus 2, remainder 1);`,
+
+		`CREATE TABLE cust_arr_large PARTITION OF cust_active FOR VALUES FROM (101) TO (MAXVALUE) PARTITION BY HASH(id);`,
+		`CREATE TABLE cust_part21 PARTITION OF cust_arr_large FOR VALUES WITH (modulus 2, remainder 0);`,
+		`CREATE TABLE cust_part22 PARTITION OF cust_arr_large FOR VALUES WITH (modulus 2, remainder 1);`,
```

Comment:
one more case to add: case-sensitive partitioned table / partitions names

Reply:
done